ui/retribution_paladin/sim.ts

```diff
@@ -140,18 +143,22 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecRetributionPaladin, {
 
 	presets: {
 		rotations: [
+			...Presets.APLPresets[Phase.Phase4],
+			PresetUtils.makePresetAPLRotation('P4 Ret-Exodin', APLP4RetExodinJson),
```

Comment:
Can we do this inside the `presets.ts` file along with the other rotation?

Reply:
We can. I cant put it in the phase array though

Reply:
Oh really? Why not?

Reply:
it's addressed by the phase variable, so I would have to put it in phase 5 right?